README.md

```diff
@@ -210,6 +210,92 @@ This app uses [RSpec](http://rspec.info/) for Rails tests.
 
     rake parallel:spec
 
+## Required environment variables for
```

Comment:
My 1st thought to be `for:` 

* The `first thing`
* The `second thing`
etc...

But I really like @threedaymonk 's suggestion. :+1: